### PR TITLE
Duo Connect v1.1.0.

### PR DESCRIPTION
**Pull request by notpeter**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/duo-connect.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'duo-connect' do
+  version '1.1.0'
+  sha256 '3cfefc97492681853fe310fc1622a095a6befc289151b7079e8ef3e406822e92'
+
+  # dl.duosecurity.com was verified as official when first introduced to the cask
+  url "https://dl.duosecurity.com/DuoConnect-#{version}.pkg"
```
Comment (vitorgalvao) [inline Casks/duo-connect.rb]: Where did you find this particular URL?

Comment (notpeter) [reply Casks/duo-connect.rb]: [Duo publishes checksums](https://duo.com/docs/checksums#duoconnect-for-macos) and if you and if you click 3cfefc97492681853fe310fc1622a095a6befc289151b7079e8ef3e406822e92 it links to DuoConnect-1.1.0.pkg.

That said, they support both versioned filenames and a `-latest` should I be using DuoConnect-latest.pkg with `sha256 :no_check` instead?

* https://dl.duosecurity.com/DuoConnect-1.0.0.pkg
* https://dl.duosecurity.com/DuoConnect-1.1.0.pkg
* https://dl.duosecurity.com/DuoConnect-latest.pkg

Comment (vitorgalvao) [reply Casks/duo-connect.rb]: > That said, they support both versioned filenames and a `-latest` should I be using DuoConnect-latest.pkg with `sha256 :no_check` instead?

No, you made the right choice. I was looking for a way to grab an appcast.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/duo-connect.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  appcas 'https://duo.com/docs/duoconnect-notes'
```
Comment (notpeter) [inline Casks/duo-connect.rb]: @vitorgalvao Should this have been `appcast`?

Comment (vitorgalvao) [reply Casks/duo-connect.rb]: Yes. It has already been fixed.

**Comment by core-code** [comment]
@vitorgalvao this cask doesn't have a app or pkg stanza. does it even *do* something when installed?

**Comment by vitorgalvao** [comment]
@core-code It doesn’t! What an oversight. `audit` was supposed to enforce some stanzas, if I recall correctly. Unsure why it didn’t catch it.

Even more surprising is how no one has complained, considering this has 38 downloads since inclusion.

Fixing on https://github.com/Homebrew/homebrew-cask/pull/82203.

**Pull request merged**